### PR TITLE
chore: add whiskers check to ensure themes are up to date

### PR DESCRIPTION
**Pull request by davfsa**

### CODE REVIEW

**Comment by sgoudham** [comment]
FWIW, @backwardspy and I are interested in making an organisation wide whiskers github action to avoid duplicating this type of workflow across every repo.

Also, I believe you'll want to pull down the same version as specified in your [fleet.tera](https://github.com/catppuccin/fleet/blob/81196ae2f686b3c13ea4e2984f0cd70278b0a3eb/fleet.tera#L4) file, otherwise you may run into version discrepencies that fail the workflow in the future.

File: .github/workflows/whiskers-check.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Fetch latest whiskers
+        run: |
+          curl -LsSf https://github.com/catppuccin/whiskers/releases/latest/download/whiskers-x86_64-unknown-linux-gnu > whiskers
```
Comment (sgoudham) [inline .github/workflows/whiskers-check.yml:16]:
```suggestion
          curl -LsSf https://github.com/catppuccin/whiskers/releases/download/v2.4.0/whiskers-x86_64-unknown-linux-gnu > whiskers
```

**Review by sgoudham — CHANGES_REQUESTED** [review]
As per my previous comment

**Comment by davfsa** [comment]
> FWIW, @backwardspy and I are interested in making an organisation wide whiskers github action to avoid duplicating this type of workflow across every repo.

Oh thats nice! Once I was done with this one (for some reason it won't run :pensive:) I was planning on messaging you to ask if you were interested in making org level (would need quite a lot of changes i believe, but could be doable)

> Also, I believe you'll want to pull down the same version as specified in your [fleet.tera](https://github.com/catppuccin/fleet/blob/81196ae2f686b3c13ea4e2984f0cd70278b0a3eb/fleet.tera#L4) file, otherwise you may run into version discrepencies that fail the workflow in the future.

Would require extracting from the tera file, so Ill have a look into how to do that!

**Comment by sgoudham** [comment]
> Would require extracting from the tera file, so Ill have a look into how to do that!

I can technically make Renovate do it automatically, suppose you can also extract it from the tera file but I'm not too fussed about creating a generalisable solution.

**Review by sgoudham — APPROVED** [review]

**Pull request merged**